### ISSUE TITLE
ImageBitmap not handled well.

### ISSUE THREAD

**Issue by jrmuizel** [kdashg/canvas-rr#10]
The following html:
```html
<!DOCTYPE html>
<html>

<body>
    <canvas id="myCanvas" width="300" height="300"></canvas>

    <script>
    function draw() {
        const canvas = document.getElementById('myCanvas');
        const ctx = canvas.getContext('2d');

        const img = new Image();
        img.src = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUA' +
          'AAAFCAYAAACNbyblAAAAHElEQVQI12P4' +
          '//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==';

        img.onload = function() {
            createImageBitmap(img).then(function(imageBitmap) {
                ctx.drawImage(imageBitmap, 0, 0);
            }).catch(function(error) {
                console.error('Error creating ImageBitmap:', error);
            });
        };
    }
    requestAnimationFrame(draw);
    </script>
</body>
</html>

```

records as:
```json
{
"version": 5,
"elem_info_by_key": {
   "$1": {
      "type": "HTMLCanvasElement",
      "width": 300,
      "height": 300
   }
},
"frames": [
   [
      ["$1","getContext",["\"2d"],"$2"]
   ],[
      ["$2","drawImage",["$3",0,0]]
   ],[
   ]
],
"snapshots": {}
}
```
and gives the following warning:
```
[LogCanvas@null] Warning: Unrecognized type "ImageBitmap" in snapshot_str for drawImage.0:  
ImageBitmap { width: 5, height: 5 }
```

**Comment by jrmuizel** [comment]
Actually this might be fixed by fccccb49a456d29d388a6b47cf85896902e9829c. I don't think I had that commit when I ran into this.

**Comment by jrmuizel** [comment]
I'm confusing ImageData with ImageBitmap. ImageBitmap still seems broken.
